plugins/database/src/db_plugin.cpp

```diff
@@ -8907,9 +8945,12 @@ irods::error db_set_avu_metadata_op(
 
     if ( status <= 0 ) {
         if ( status == CAT_NO_ROWS_FOUND ) {
-            /* Need to add the metadata */
-            status = chlAddAVUMetadata( _ctx.comm(), 0, _type, _name, _attribute,
-                                        _new_value, _new_unit );
+            // Need to add the metadata.
+            //
+            // TODO This function used to be executed with _admin_mode set to zero.
+            // Are the semantics identical when _cond_input holds ADMIN_KW?
+            status = chlAddAVUMetadata( _ctx.comm(), _type, _name, _attribute,
+                                        _new_value, _new_unit, _cond_input );
```

Comment:
remove TODO comment.

if the admin keyword is set, all calls that deal with permissions should honor it.